doc/attr.xml

```diff
@@ -1068,6 +1067,95 @@ gap> ArticulationPoints(D);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="StrongOrientation">
+<ManSection>
+  <Oper Name="StrongOrientation" Arg="D"/>
+  <Attr Name="StrongOrientationAttr" Arg="D"/>
+  <Returns>A digraph or <K>fail</K>.</Returns>
+  <Description>
+    A <E>strong orientation</E> of a connected symmetric digraph <A>D</A> (if
+    it exists) is a strongly connected subdigraph <C>C</C> of <A>D</A> such
+    that for every edge <C>[u, v]</C> of <A>D</A> either <C>[u, v]</C> or
+    <C>[v, u]</C> is an edge of <C>C</C> but not both.  By Robbin's Theorem,
+    only bridgeless digraphs (see <Ref Prop="IsBridgelessDigraph"/>) admit a
```

Comment:
Is this an if and only if? If so, I think this should be said more explicitly.

Reply:
Yup it is, made explicit